composer.json

```diff
@@ -10,6 +10,7 @@
     "require": {
         "php": "^7.1.3",
         "backpack/crud": "4.0.*",
+        "backpack/permissionmanager": "^5.0",
         "barryvdh/laravel-elfinder": "^0.4.5",
         "fideloper/proxy": "^4.0",
         "laravel/framework": "5.8.*",
```

Comment:
Big one: I recommend updating to Laravel 6.0. It's the current "LTS" version, and they've just ended security patch support for 5.8. 

You probably won't use most of the 6.0 features, but it's worth it for the security support, and doing it early makes for a much easier upgrade process. (Note, Laravel 7 just came out, but I suggest sticking with 6 as the LTS version).

I'd also add "Ignition" (`facade/ignition`) - it comes default with Laravel 6, but you don't get it on upgrade. It's a huge upgrade on the old Laravel error screen, as it can suggest fixes for common errors, and lets you do things like jump straight to the exception point in your editor of choice (see the [config options here](https://flareapp.io/docs/ignition-for-laravel/configuration)).